### DISCUSSION TITLE
Newegg problem

### DISCUSSION THREAD

**Discussion by qq442539105** [jef/streetmerchant#1403] [Help]
STORES=“newegg，amazon”
Newegg does not take effect after adding！
![st](https://user-images.githubusercontent.com/56084459/102080411-5dabb180-3e49-11eb-868c-4bc5890e0334.jpg)

**Comment by jef** [comment]
Do you have any other filters?

**Comment by FlyingToto** [comment]
this is even weirder since you can see walmart in your screenshot while your filter is only "newegg, amazon"
double check that your config file is not saved with .txt extension by mistake... also not sure if it is related I don't separate stores with a space, only a comma so this `"newegg,amazon" `instead of your `"newegg, amazon"`...
